client/my-sites/theme/theme-related-themes-card/index.jsx

```diff
@@ -0,0 +1,86 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import i18n from 'lib/mixins/i18n';
+import SectionHeader from 'components/section-header';
+import { getDetailsUrl } from 'my-sites/themes/helpers';
+
+var ThemeRelatedThemesCard = React.createClass( {
+
+	propTypes: {
+		currentTheme: React.PropTypes.string.isRequired
+	},
+
+	getRelatedThemes() {
+		var themes = [
```

Comment:
this can be a `const` instead of a `var`:
https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations